distribution.md

```diff
@@ -155,3 +155,73 @@ Where:
     * contain an array of deprecated APIs:
       * should only contain `0.x` or major versions
       * should only contain APIs that correspond to a spec release
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A Builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting the builder SHOULD allow users to configure the build environment with a provided builder.
+
+#### General Requirements
+
+The Builder image MUST contain an implementation of the [lifecycle](#lifecycle), and [build-time](#build-image) environment and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process. 
+
+#### Environment Variables
+
+A Builder MUST be an extension of a Build Image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+#### Labels
+
+A Builder MUST be an extension of a Build Image, and MUST retain all the specified Labels set on the original build image, as specified in the Build Image specifications.
+
+A Builder image MUST contain an implementation of the [lifecycle](#lifecycle), and MUST retain all the specified Labels set on the original Lifecycle image, as specified in the lifecycle distribution specifications.
```

Comment:
@jkutner what I meant by that line is that the Builder image (which MUST contain the lifecycle implementation) must also have the labels specified on the lifecycle image. So if I have to distribute a Builder Image, it must have all the labels set on lifecycle because it contains the lifecycle implementation.

Reply:
@jimil749 are those labels spec'ed somewhere?

Reply:
Yep, in the lifecycle distribution specification. https://github.com/buildpacks/spec/pull/264

Reply:
The goal is to reference the buildpack components i.e. build image, run image, buildpacks and lifecycle from the builder image.

Reply:
@jkutner do you want any changes for this?